src/static/js/wishlist.js

```diff
@@ -91,3 +91,45 @@ setTimeout(() => {
   document.querySelectorAll('.upForm').forEach(element => { listen(element, 'up') })
   document.querySelectorAll('.downForm').forEach(element => { listen(element, 'down') })
 }, 0)
+
+function priceToFloat(price) {
+  let cleanPrice = price
+      .replace(/[^\d.,]/g, '')  // Remove everything except digits, periods, and commas
+      .replace(/,/g, '');       // Remove commas to standardize the number
+  // If there's more than one decimal separator (.), treat the last one as the decimal point
+  if ((cleanPrice.match(/\./g) || []).length > 1) {
+      cleanPrice = cleanPrice.replace(/\.(?=.*\.)/, '');  // Remove the first period if there are multiple
+  }
+  return parseFloat(cleanPrice);
+}
+
+document.addEventListener('DOMContentLoaded', function(){
+  let sortOrderPrice = 'asc';
+  const priceArrow = document.getElementById('price-arrow');
+  const sortByPrice = document.getElementById('sort-price');
+
+  function updateArrow(column, order) {
+    if (column === 'price') {
+      priceArrow.className = order === 'asc' ? 'fas fa-arrow-up' : 'fas fa-arrow-down';
+    }
+  }
+
+  sortByPrice.addEventListener('click', function(e){
```

Comment:
Same arrow function request

Reply:
I'm not sure what you mean by this. Can you elaborate?

Reply:
Resolved with https://github.com/Wingysam/Christmas-Community/pull/140/commits/4d84b2bc9988f4b8e8843230b0cf127740a246b3